core/src/test/java/org/aerogear/mobile/core/http/OkHttpServiceModuleTest.java

```diff
@@ -43,4 +43,26 @@ public void testGet() {
         response.waitForCompletionAndClose();
     }
 
+    @Test
+    public void testErrorHandler() {
```

Comment:
Please @pb82 rename this method accordingly with what the test case is about

Reply:
But it is about testing the error handler?